WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostRenderer.java

```diff
@@ -307,6 +310,9 @@ private void replaceIframeTag(final String tag, final String src) {
      * returns the full content, including CSS, that will be shown in the WebView for this post
      */
     private String formatPostContentForWebView(final String content, boolean renderAsTiledGallery) {
```

Comment:
It looks like the most reliable technique is to check whether the `format` field of the reader post is "gallery."

Unfortunately, we haven't been storing `format` - but as luck would have it, I added it just today in PR #4316 along with an <a href="https://github.com/wordpress-mobile/WordPress-Android/pull/4316/files#diff-0bd1e069dded619660267aa1a74332cfR367">isGallery()</a> method to check whether a post contains a gallery.

So once that PR is merged, you can use that method.